wiki/Legal/Copyright/ja.md

```diff
@@ -0,0 +1,63 @@
+---
+legal: true
+---
+
+# osu!著作権ポリシー
+
+osu!はコミュニティ志向の強いゲームです。全てのゲーム譜面（ビートマップ）はコミュニティのメンバーによって作成・アップロードされているため、関連する著作権保有者から正式な許可を得られないまま作成されることが頻繁に発生しています。osu!ではスポンサー広告や、アップロードされたコンテンツからの収益が発生しないよう徹底しており、ゲームの資金はすべてユーザーの寄付によって賄われています。
+
+私たちが受け取った寄付金の大部分は、[音楽ライセンス](/wiki/Legal/Music_licensing)への再投資に充てられています。ライセンスに関するお問い合わせは、[music@ppy.sh](mailto:music@ppy.sh) までお願いします。　
+
+それでも解決しない場合、osu!は、デジタルミレニアム著作権法（17 U.S.C. 512）に基づき、著作権侵害に関する以下の方針を採択します。
+
+osu!の指定侵害通知受領代理人（「指定代理人」）の連絡先は、このポリシーの末尾に記載されており、アメリカ合衆国著作権局（<https://www.copyright.gov/dmca-directory/>）に登録されています。osu!は著作権とその他の知的所有権を非常に重要視しています。
+
+osu!の著作権ポリシー:
+
+1. 第三者の著作権を侵害する素材を含むと真摯に判断したコンテンツへのアクセスを迅速にブロックまたは削除すること、および
+2. 侵害を繰り返すユーザーに対しては、osu!アカウントを削除し、サービスを停止します。
+
+## 著作権侵害の報告
+
+osu!のウェブサイトまたはサービスを通じ存在しているコンテンツ、あるいはアクセス可能なコンテンツがあなたの著作権を侵害していると思われる場合、以下の情報を含む著作権侵害の申し立て通知を以下に示す指定代理人にお送りください（これらの要件を確認するには、弁護士に相談するか合衆国法典17 U.S.C. 512（c）を参照してください）。
+
+1. 侵害されたとされる著作権の所有者の代理として行動することを承認された人物の直筆署名または電子署名。
+2. 侵害されたと主張する著作物の特定、または単一のオンラインサイトにおける複数の著作物が単一の通知の対象となる場合は、当該サイトにおける当該著作物の代表的なリスト。
+3. 侵害行為または侵害行為の対象であると主張され、削除またはアクセス不能にされるべき素材の特定、およびosu!がその素材を特定するために合理的に十分な情報。
+4. 通知者の連絡先(住所、電話番号、可能であればemailアドレスなど)
+5. 申し立ての対象となる素材のosu!における使用が、著作権者、その代理人、または法律によって認められていないと通知者が誠意を持って確信していることを示す声明。
+6. 通知の情報に偽りがないこと、偽証が処罰の対象であることを承知した上で、通知当事者が侵害されたとされる著作権の所有者の代理として行動することを承認されていることを示す声明。　
+
+著作権侵害を主張する完全かつ適切な通知を指定代理人が受け取った場合、またはosu!サービス上のファイルに著作権を侵害する素材が含まれている可能性があるとosu!が誠意を持って判断するに至った場合：
+
+osu!の著作権ポリシー:
+
+1. 侵害の申し立て通知で特定されたコンテンツを削除、またはアクセス不能にします。
+2. コンテンツ提供者、メンバーまたはユーザーに、コンテンツへのアクセスを削除または無効にしたことを通知すること、および
+3. 適切な状況において、侵害を繰り返す加入者およびアカウント保有者によるosu!利用を終了させます。
+
+## 反対通知の提出
+
+コンテンツ提供者、メンバーまたはユーザーが、削除されたまたはアクセス不能にされたコンテンツが侵害ではないと信じる場合、またはコンテンツ提供者、メンバーまたはユーザーが、著作権者や著作権者の代理人から又は法律の規定により当該コンテンツを投稿して利用する権利があると信じているとき、コンテンツ提供者、メンバーまたはユーザーは、以下の情報を含む反対通知書を以下に示す指定代理人に送付することができるものとします。
+
+1. コンテンツ提供者、メンバー、ユーザーの直筆または電子的な署名。
+2. 削除された、またはアクセスが無効にされたコンテンツと、削除される前またはアクセスが無効にされる前に当該素材が表示されていた場所を特定できるもの。
+3. 偽証罪の罰則のもと、コンテンツ提供者、メンバー、またはユーザーが、当該素材が削除または使用不能にされたことは、錯誤または誤認の結果であると誠意を持って確信しているという声明文、および　
+4. コンテンツ提供者、メンバーまたはユーザーの名前、住所、電話番号、および可能であれば電子メールアドレス、および当該個人または団体がコンテンツ提供者、メンバーまたはユーザーの住所が所在する司法地区の連邦地方裁判所（コンテンツ提供者、メンバーまたはユーザーの住所が米国外にある場合は、カリフォルニア州ロサンゼルス郡の管轄）に同意し、当該個人または団体は侵害の疑いについて申し立て通知を行った者またはその代理人からの送達を受け入れるという声明文。
```

Comment:
Same as L41.
Also, I think it can separate into 2.
Example: `4. コンテンツ提供者、メンバーまたはユーザーの名前、住所、電話番号、および可能であれば電子メールアドレス
5.当該個人または団体がコンテンツ提供者、メンバーまたはユーザーの住所が所在する司法地区の連邦地方裁判所（米国外の場合は、カリフォルニア州ロサンゼルス郡の管轄）に同意し、当該個人または団体は侵害の疑いについて申し立て通知を行った者またはその代理人からの送達を受け入れるという声明文。`